include/lsst/pex/logging/BlockTimingLog.h

```diff
@@ -35,7 +35,6 @@
 
 #include <sys/time.h>
 #include <sys/resource.h>
-#include <boost/scoped_ptr.hpp>
```

Comment:
Do you also need to include <memory> here? I can't see the whole file in this diff, so maybe you already have done so.